assignments/arrays.js

```diff
@@ -66,31 +66,71 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
+for (let i = 0; i < inventory.length; i++) {
+	console.log(`Car ${inventory[i]["id"]} is a ${inventory[i]["car_year"]} ${inventory[i]["car_make"]} ${inventory[i]["car_model"]}`);
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
 console.log();
 
+let lastCar = 0;
+lastCar = inventory[inventory.length - 1]["car_make"] + " " + inventory[inventory.length - 1]["car_model"];
```

Comment:
You can just assign lastCar = inventory[inventory.length-1] and written out.

console.log(lastCar.car_make + " " + lastCar.car_model);

Just a suggestion.